src/styles/_search-bar.scss

```diff
@@ -0,0 +1,45 @@
+#search-bar-div {
```

Comment:
See if you can match the border style of the `<header>` element in `_header-title.scss` to make the UI a little more consistent visually.

Reply:
@jimboweb not sure if you saw this comment or not?

Reply:
Matched the style

Reply:
@jimboweb Also perhaps add `background-clip: padding-box;` to keep is more consistent.